vscode-dotnet-runtime-extension/CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning].
 
 ## [Unreleased]
 
+## [2.1.2] - 2024-08-01
+
+Adds the ability to uninstall a particular runtime.

```

Comment:
I am looking at https://github.com/dotnet/vscode-dotnet-runtime/pull/1894, did this actually made it in?

Reply:
Not yet, I am doing the branding before the changes go in with knowing what will go into the version in advance. I have full control over the release process so I can pretty much determine what will be in the release/when will the release be.